drivers/SCsub

```diff
@@ -9,15 +9,19 @@ SConscript("unix/SCsub")
 SConscript("windows/SCsub")
 
 # Sounds drivers
-SConscript("alsa/SCsub")
-SConscript("coreaudio/SCsub")
-SConscript("pulseaudio/SCsub")
+if env["platform"] in ["ios", "macos"]:
+    SConscript("coreaudio/SCsub")
+
+if env["platform"] == "linuxbsd":
+    if env["alsa"]:
+        SConscript("alsa/SCsub")
+    if env["pulseaudio"]:
+        SConscript("pulseaudio/SCsub")
+
 if env["platform"] == "windows":
+    if env["xaudio2"]:
+        SConscript("xaudio2/SCsub")
     SConscript("wasapi/SCsub")
```

Comment:
I think it's better this way, as no empty files will be compiled. Maybe it would be better to add checks like this to other drivers?